### PR TITLE
feat: refresh chart to use latest connect version

### PR DESCRIPTION
**Pull request by ooesili**
After this PR is merged, I will move the contents of this repo to [the main charts repo](https://github.com/redpanda-data/helm-charts/tree/main/charts) to keep it in line with everything else. After that we should archive this repo and put a notice pointing to the new location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-Benthos is a high performance and resilient stream processor, able to connect various sources and sinks in a range of brokering patterns and perform hydration, enrichments, transformations and filters on payloads.
+Redpanda Connect is a high performance and resilient stream processor, able to connect various sources and sinks in a range of brokering patterns and perform hydration, enrichments, transformations and filters on payloads.
 
-This Helm Chart deploys a single Benthos instance in either streams mode or standalone.
+This Helm Chart deploys a single Redpanda Connect instance in either streams mode or standalone.
 
 ## Installation
 ```bash
-helm repo add benthos https://benthosdev.github.io/charts/
+helm repo add redpanda https://benthosdev.github.io/charts/
```
Comment (mihaitodor) [inline README.md:13]: Should `benthosdev` change to `redpanda-data`? Also, should it be `helm repo add redpanda-connect` instead?

File: values.yaml [hunk 2]
```diff
@@ -117,7 +117,7 @@ env: []
 envFrom:
   []
   # - secretRef:
-  #     name: benthos
+  #     name: redp
```
Comment (mihaitodor) [inline values.yaml]: redpanda-connect?

**Review by mihaitodor — APPROVED** [review]
LGTM, just a few small nits 🐑 🚀

File: README.md (hunk 1 above)
Comment (ooesili) [reply README.md:13]: This is going to change to `helm repo add redpanda https://charts.redpanda.com` after this moves to the main charts repo. The existing documentation for the main chart reop uses `redpanda` as the repository name so I wanted to match that usage.

File: values.yaml (hunk 2 above)
Comment (ooesili) [reply values.yaml]: Whoops, fixed

File: README.md (hunk 1 above)
Comment (mihaitodor) [reply README.md:13]: Oh, OK. That makes sense, thank you!

**Comment by ooesili** [comment]
We're going to move this chart somewhere else...

**Pull request closed without merging**